test/uk/gov/hmrc/economiccrimelevyreturns/connectors/EclCalculatorConnectorSpec.scala

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.economiccrimelevyreturns.connectors
+
+import org.mockito.ArgumentMatchers
+import org.mockito.ArgumentMatchers.any
+import uk.gov.hmrc.economiccrimelevyreturns.base.SpecBase
+import uk.gov.hmrc.economiccrimelevyreturns.generators.CachedArbitraries._
+import uk.gov.hmrc.economiccrimelevyreturns.models.{CalculateLiabilityRequest, CalculatedLiability}
+import uk.gov.hmrc.http.HttpClient
+
+import scala.concurrent.Future
+
+class EclCalculatorConnectorSpec extends SpecBase {
+  val mockHttpClient: HttpClient = mock[HttpClient]
+  val connector                  = new EclCalculatorConnector(appConfig, mockHttpClient)
+  val eclCalculatorUrl           = "http://localhost:14010/economic-crime-levy-calculator"
```

Comment:
Shouldn't this obtain the base Url from AppConfig?

Reply:
it could be obtained from app config but it's hard coded like this in the test so that if the url is changed then the test will fail